src/ReplicatedStorage/Client/InstanceComponents/Misc/CameraLook.lua

```diff
@@ -0,0 +1,46 @@
+local Players = game:GetService("Players")
+local RunService = game:GetService("RunService")
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Trove = require(ReplicatedStorage.Packages.Trove)
+
+local CameraLook = {}
+CameraLook.__index = CameraLook
+
+function CameraLook.new(instance: Instance)
+	local self = setmetatable({}, CameraLook)
+
+	self.Instance = instance
+	self._trove = Trove.new()
+
+	local humanoid = instance:FindFirstChildOfClass("Humanoid")
+	local rootPart = humanoid.RootPart
+
+	local player = Players:GetPlayerFromCharacter(instance)
+	local mouse = player:GetMouse()
+	self._trove:Add(RunService.Stepped:Connect(function()
```

Comment:
This should use Trove:Connect()

Reply:
I agree

Reply:
> I agree

not allowed (agreeing may be against the `CONTRIBUTING.md` (TBD))

Reply:
somehow I made this change but you requested review on an older version or something I think, I have no idea how this happened but marking as resolved since it should be good already